test/specs/components/reply-actions.spec.js

```diff
@@ -21,23 +36,19 @@ describe('QuickReplies Component', () => {
             text: 'choice 2',
             payload: 'payload 2',
             iconUrl: 'http://some-url/'
+        },
+        {
+            text: 'Send Location',
+            type: 'locationRequest'
         }
     ];
 
     beforeEach(() => {
-        mockedStore = createMockedStore(sandbox, {
-            app: {
-                settings: {
-                    web: {
-                        accentColor: '',
-                        isAccentColorDark: true
-                    }
-                }
-            }
-        });
         sandbox.stub(conversationService, 'sendMessage');
+        sandbox.stub(conversationService, 'sendLocation');
+        navigator.geolocation = true;
```

Comment:
We should keep the default value and reassign it in `afterEach`. It could have unwanted side effects on future tests.